src/components/VideoItem.vue

```diff
@@ -10,13 +10,25 @@
                 },
             }"
         >
-            <img :height="height" :width="width" class="w-full" :src="video.thumbnail" alt="" loading="lazy" />
+            <img
+                :style="`max-height: ${height}px; max-width: ${width}px`"
+                class="w-full"
+                :src="video.thumbnail"
+                :alt="video.title"
+                loading="lazy"
+            />
             <div class="relative text-sm">
                 <span
                     class="thumbnail-overlay thumbnail-right"
                     v-if="video.duration > 0"
                     v-text="timeFormat(video.duration)"
                 />
+                <span class="thumbnail-overlay thumbnail-right" v-if="video.duration <= 60" v-text="`SHORTS`" />
```

Comment:
This should ideally be translatable.

Reply:
@FireMasterK done, please check https://github.com/TeamPiped/Piped/pull/1141/commits/00aa83b802cf19c469a64f1792a70177d28fb64b

Reply:
ok, found it, check https://github.com/TeamPiped/Piped/pull/1141/commits/18385ec7759988268dce7eef891ab5a507f7d492

Reply:
## Final design 

![image](https://user-images.githubusercontent.com/11899221/174738344-78998fe6-a7cf-45a7-8c81-d9011113a10d.png)

Reply:
Hey, can we use the shorts logo instead of text? https://en.wikipedia.org/wiki/File:Youtube_shorts_icon.svg

Reply:
@FireMasterK I hope we're good to go